### PR TITLE
Rerun feeder has required permissions

### PR DESCRIPTION
**Pull request by marchuffnagle**

### CODE REVIEW

**Comment by matthewhanson** [comment]
The changes look fine, but the tests are failing due to a separate issue, looks like some fixture data might need to be updated.... @jkeifer ?

**Comment by jkeifer** [comment]
@matthewhanson The cli tests maintain a list of files that should be created by `cirrus build` and their checksums. The exception is the generated `serverless.yml` file, which is fully checked in as a fixture. Between these two sources of truth, the tests can show what has changed. It's useful to ensure that we haven't had a regression or introduced any unexpected behavior in the build, but it does have the disadvantage that the tests need to be updated if any of the build sources have been updated (I struggled to come up with how to test this well and this was the best idea I had, but I am open to better ones if anyone has suggestions).

In this case, as the rerun feeder perms have been updated, the `serverless.yml` fixture is now different than the build result. Here is the relevant section of the test output:

```
Files in missing from build: []
Files added in build: []
Files different in build: ['serverless.yml']
--- expected serverless.yml
+++ generated serverless.yml
@@ -303,6 +303,23 @@
           - ec2:DescribeNetworkInterfaces
           - ec2:DeleteNetworkInterface
         Resource: '*'
+      - Effect: Allow
+        Action:
+          - dynamodb:Query
+          - dynamodb:Scan
+          - dynamodb:GetItem
+          - dynamodb:BatchGetItem
+          - dynamodb:DescribeTable
+        Resource:
+          - Fn::GetAtt:
+              - StateTable
+              - Arn
+          - Fn::Join:
+              - ''
+              - - Fn::GetAtt:
+                    - StateTable
+                    - Arn
+                - /index/*
     description: Rerun items in the database
     environment:
       CIRRUS_LOG_LEVEL: DEBUG

Reference build: /home/runner/work/cirrus-geo/cirrus-geo/tests/fixture_data/build
Test build directory: /home/runner/work/cirrus-geo/cirrus-geo/tests/output/.cirrus

If all highlighted changes are expected, simply remove 
the reference_build directory and rerun the tests to update 
the reference files.
```

In short, `rm -r tests/fixture_data/build` and run `pytest`. The fixtures will be re-created because they were removed. Double-check the changes in the diff before committing.

I went ahead and updated that, as well as added the missing link reference in the changelog.
